plugins/inputs/win_perf_counters/win_perf_counters.go

```diff
@@ -429,21 +512,33 @@ func (m *Win_PerfCounters) Gather(acc telegraf.Accumulator) error {
 			}
 		}
 	}
-
-	for instance, fields := range collectFields {
+	for instance, fields := range collectedFields {
 		var tags = map[string]string{
 			"objectname": instance.objectname,
 		}
 		if len(instance.instance) > 0 {
 			tags["instance"] = instance.instance
 		}
-		acc.AddFields(instance.name, fields, tags, timestamp)
+		if len(hostCounterInfo.computerTag) > 0 {
```

Comment:
Instead, check if `.computer` is empty and set to localhost, otherwise set to the `source` (computer->source) tag to the value of `.computer`